openshift-install-powervs

```diff
@@ -1339,6 +1357,16 @@ function process_varfile {
   done
 }
 
+#-------------------------------------------------------------------------
+# Flavor
+#-------------------------------------------------------------------------
+function validate_flavor {
+   [[ -z "$flavor" ]] &&  error "The -flavor option is not provided"
+
+   flavor_list=($(ls automation/compute-vars/))
+   (for i in "${flavor_list[@]}"; do [[ "$i" == "$flavor.tfvars" ]] && exit 0; done)  || error "The -flavor '$flavor' is not valid. Please select either ${flavor_list[*]/.tfvars/}"
```

Comment:
Instead of for loop we can use grep here.

`ls automation/compute-vars/ | grep "$flavor.tfvars"`

Reply:
Used '-e' flag since grep would print the result.